app/forms/idv/in_person/address_form.rb

```diff
@@ -38,13 +38,22 @@ def submit(params)
       def consume_params(params)
         params.each do |key, value|
           raise_invalid_address_parameter_error(key) unless ATTRIBUTES.include?(key.to_sym)
-          send("#{key}=", value)
+
+          if key == 'same_address_as_id'
+            send("#{key}=", infer_boolean_type(value))
+          else
+            send("#{key}=", value)
+          end
```

Comment:
With only 6 attributes, I think that unrolling the loop and removing the constant entirely is a viable option that's just as maintainable. I think the `params.permit()` is not needed since the form is pulling specific keys out of the params has anyways (rails strong parameters exists to stop handing an entire hash with unknown keys directly to the database, but since this form pull specific keys out anyways, it's not needed)

```suggestion
  @state = params[:state]
  @zipcode = params[:zipcode]
  @city = params[:city]
  @address1 = params[:address1]
  @address2 = params[:address2]
  @same_address_as_id = ActiveRecord::Type::Boolean.new.cast(params[:same_address_as_id])
```